lib/enigma.rb

```diff
@@ -1,16 +1,43 @@
+require "date"
 require "./lib/message_processor"
 class Enigma
 
   def output_hash(processed_message, key, date, action = :encryption)
     Hash[action, processed_message, :key, key, :date, date]
   end
 
-  def encrypt(input_message, key, date)
+  def random_number
+    (0..9).to_a.sample(5).shuffle.join
```

Comment:
Need to update this, it will never return duplicate numbers in the string. such as "01122"